stdlib/function.ncl

```diff
@@ -7,5 +7,35 @@
     const : forall a b. a -> b -> a
     | doc "The const function, results in the first argument."
     = fun x y => x,
+
+    compose : forall a b c. (b -> c) -> (a -> b) -> (a -> c)
+    | doc "Function composition, right to left."
```

Comment:
Could be useful to write down how it acts on an element: `compose f g x = f (g x)` in the documentation